supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -290,7 +295,7 @@ object TesAsyncBackendJobExecutionActor {
           responseLogs <- t.logs
           startTime <- responseLogs.headOption.map(_.start_time)
           vmCost <- responseLogs.headOption.map(_.metadata.flatMap(_.get("vm_price_per_hour_usd")))
-          tesVmCostData = TesVmCostData(startTime, vmCost)
+          tesVmCostData = TesVmCostData(startTime, None, vmCost)
```

Comment:
```suggestion
          // NB: End time is omitted here so we don't keep polling for it while the task runs. It will be acquired with a separate request when the task completes. 
          tesVmCostData = TesVmCostData(startTime, None, vmCost)
```